doc/include/api_embedc.inc

```diff
@@ -263,42 +194,33 @@ be [const Tcl_Interp*]. In that case the argument in question is
 [comment ---------------------------------------------------------------------]
 [call [cmd ::critcl::cproc] [arg name] [arg arguments] [arg resulttype]]
 
-This variant of [cmd critcl::cproc] assumes that the functionality to
-connect to is implemented by the C function [arg name] which has the
-signature described by the [arg arguments] and [arg resulttype].
+As documnented below, but used when the C function named [arg name] already
```

Comment:
`documnented` -> `documented`

Reply:
Done with commit 29380521d0873ea1dcc5f8cdff6742b20764473f (Branch `documentation` in my repo, not seen by the PR).